plugins/jenkins-backend/src/service/jenkinsInfoProvider.ts

```diff
@@ -84,7 +84,7 @@ export class JenkinsConfig {
         baseUrl: c.getString('baseUrl'),
         username: c.getString('username'),
         apiKey: c.getString('apiKey'),
-        headers: c.getOptional('headers'),
+        extraRequestHeaders: c.getOptional('extraRequestHeaders'),
         crumbIssuer: c.getOptionalBoolean('crumbIssuer'),
```

Comment:
Do you want to add these two to the [config schema](https://github.com/backstage/backstage/blob/c02fda193d4d582ff955e007e1eebb3afe3d1206/plugins/jenkins-backend/config.d.ts#L18-L47) as well in a followup?